pindel/pindel.cwl

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "pindel v0.2.5b8"
+baseCommand: ["/usr/bin/pindel", "-i", "pindel.config"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "mgibio/pindel-cwl:v0.2.5b8"
+    - class: InitialWorkDirRequirement
+      listing:
+          - entryname: 'pindel.config'
+            entry: |
+                $(inputs.normal_bam.path) 400    normal
```

Comment:
I will change to NORMAL TUMOR. 400 is the hardcoded value in GMS pindel run. It can be optional if it varies.